src/braket/circuits/braket_program_context.py

```diff
@@ -134,7 +134,7 @@ def add_result(self, result: Results) -> None:
         """
         self._circuit.add_result_type(braket_result_to_result_type(result))
 
-    def handle_parameter_value(self, value):
+    def handle_parameter_value(self, value: Any) -> Any:
```

Comment:
This should actually be `Union[float, Expr] -> Any` in the base class and `Union[float, Expr] -> Union[float, FreeParameterExpression]` here.

I'll update the base class in the local sim

Reply:
(don't block merging this PR on this, I can update it separately)

Reply:
@ajberdy I think this will also fail linters for the description

Reply:
It gets its docstring from the base class in the default simulator. Wouldn't hurt to give a more specific one here though